python/pyspark/sql/tests/arrow/test_arrow_python_udf.py

```diff
@@ -243,6 +273,91 @@ def test_udf_use_arrow_and_session_conf(self):
                 udf(lambda x: str(x), useArrow=False).evalType, PythonEvalType.SQL_BATCHED_UDF
             )
 
+    def test_udf_use_arrow_and_session_conf(self):
+        self._check_udf_use_arrow_and_session_conf()
+
+
+@unittest.skipIf(
+    not have_pandas or not have_pyarrow, pandas_requirement_message or pyarrow_requirement_message
+)
+class ArrowPythonUDFLegacyTestsMixin(BaseUDFTestsMixin):
```

Comment:
TODO: also add parity tests in `pyspark.sql.tests.connect.arrow.test_parity_arrow_python_udf`